build/MSBuildExtensions.targets

```diff
@@ -99,7 +99,7 @@
     <ItemGroup>
       <_NETStandardLibraryVersions Include="@(PackageDefinitions->'%(Version)')"
                                    Condition="%(PackageDefinitions.Name) == 'NetStandard.Library'" />
-      <_NETCorePlatformsImplicitPackageVersion Include="@(PackageDefinitions->'%(Version)')"
+      <_NETCorePlatformsLibraryVersions Include="@(PackageDefinitions->'%(Version)')"
```

Comment:
Library was part of the package name for the other one.

Reply:
But there is the following `<_NETStandardPackageVersion>@(_NETStandardLibraryVersions->Distinct())</_NETStandardPackageVersion>`

Reply:
I am lost, if you vote for _NETCorePlatformsLibraryVersion**s**  -> _NETCorePlatformsPackageVersion**s** i will just go this way, both way make sense

Reply:
OK, we were already inconsistent, but we should fix it.

NETStandard.**Library** -> _NETStandard**Library**PackageVersion**s**
(Microsoft.)NETCore.Platforms -> _NETCorePlatformsPackageVersion**s**

We were missing "Package" in the singular case. Library was not a synonym for package, it was coming from the package name.

Reply:
I see, I corrected _NETStandardLibraryPackageVersions as well. However there is BundledNETStandardPackageVersion (no _Library_) below, i won't change it since it will introduce a breaking change.